humble/distribution.yaml

```diff
@@ -9021,7 +9021,7 @@ repositories:
     release:
       tags:
         release: release/humble/{package}/{version}
-      url: https://github.com/tuw-robotics/tuw_geometry-release.git
+      url: https://github.com/ros2-gbp/tuw_geometry-release.git
       version: 0.1.1-1
```

Comment:
Looking at https://github.com/ros2-gbp/tuw_geometry-release/blob/master/tracks.yaml , it is clear that a release has been done to the old https://github.com/tuw-robotics/tuw_geometry-release.git which is not reflected in https://github.com/ros2-gbp/tuw_geometry-release.git (the same goes for `rolling` below).  This means that the version listed here no longer reflects what is available in the `ros2-gbp` repository.  What should we do here?

Reply:
Thanks for flagging this. I'm confident that the script would use the appropriate version when updating Rolling because it uses the rosdistro index as the source-of-truth but that doesn't cover manual releases for tracks that aren't getting migrated. So I went ahead and integrated the changes from the upstream release repository in https://github.com/ros2-gbp/tuw_geometry-release/commit/2497e69ab48833ca35c79e2f949f461886f51edb